docs/hardware_interfaces.md

```diff
@@ -0,0 +1,81 @@
+## Target TPM hardware interfaces
+
+As we are designing a universal TPM device, we are targeting multiple communication
+interfaces to talk to TPM. Our requirements are to use the most popular ones (LPC
+and SPI).
+
+The first interface, that we looking to implement is SPI. That's mostly because we
+think it is the easiest way is to start.
+
+We are planning to use lpntpm with an embedded platform as Raspberry Pi.
+Some steps were made to archive it, but we are still in the development stage.
+Our goal is to be able to use `tpm2-tools` and emulate the behavior of
+already available TPM.
+
+## Required SPI bus speed
+
+As one of our requirements is to communicate with motherboards using SPI we
+should also implement such communication.
+
+[TCG PC Client Platform TPM Profile Specification for TPM 2.0](https://trustedcomputinggroup.org/wp-content/uploads/PC-Client-Specific-Platform-TPM-Profile-for-TPM-2p0-v1p04_r0p37_pub-1.pdf)
```

Comment:
This is too raw. Please make a sentence(s) out of it. E.g. as explaiend in []() pages XXX, there are following requirements for ....